### DISCUSSION TITLE
posts/father-and-hotel/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#15] [Announcements]
# posts/father-and-hotel/

父亲和招待所

https://blog.wohin.me/posts/father-and-hotel/

**Comment by littlezhanzhan** [comment]
写的挺好，有点好奇，父母是怎么跟你解释住在招待所的意义。

**Reply by brant-ruan** [reply]
其实差不多就是“好钢用在刀刃上”的意义。不同的经济水平、视野、经历会有不同的选择吧。对于当时的我们来说，从家乡到上海，有太多的未知，所以每一步都留更多的钱给未来，即使也没有很多。
